workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandRequestSender.cs

```diff
@@ -30,6 +33,17 @@ public class WorldCommandRequestSender : RequirableBase
                     this.entityManager = entityManager;
                 }
 
+                /// <summary>
+                ///     Sends a ReserveEntityIds command request.
+                /// </summary>
+                /// <param name="numberOfEntityIds">The number of entity IDs to reserve.</param>
+                /// <param name="timeoutMillis">
+                ///     (Optional) The command timeout in milliseconds. Will default to the WorkerSDK default.
```

Comment:
for all of these ones in this file, would rephrase "Will default to the WorkerSDK default" to "If not specified, uses the WorkerSDK default"